about.css

```diff
@@ -0,0 +1,119 @@
+body {
+  font-family: 'lato', sans-serif;
+}
+
+.main-wrapper {
+  border-style: solid;
+  width: 375px;
+  height: auto;
+}
+
+.nav-bar {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  height: 80px;
+  background-color: aliceblue;
+}
+
+.hamburger {
+  color: rgb(51, 51, 51);
+  align-self: center;
+
+  /* position: absolute; */
+  top: 48px;
+  left: 20px;
+  z-index: 1;
+  font-size: 25px;
+  margin-left: 15px;
+}
```

Comment:
- [ ] Kindly correct the position of the hamburger button. It should be on the left side 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/198774492-7034e87f-d548-4110-a6e5-f3c8f4db0647.png)